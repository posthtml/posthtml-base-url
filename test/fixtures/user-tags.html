<a href="foo/bar.html">
  <img src="image1.jpg" srcset="image1-HD.jpg 2x,image1-phone.jpg 100w">
</a>

<script src="javascript.js"></script>
