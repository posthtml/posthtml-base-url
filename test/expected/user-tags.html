<a href="foo/bar.html">
  <img src="https://example.com/image1.jpg" srcset="https://example.com/image1-HD.jpg 2x, https://example.com/image1-phone.jpg 100w">
</a>

<script src="https://example.com/javascript.js"></script>
